src/GregClient/GregClient.cs

```diff
@@ -85,6 +87,40 @@ public ResponseWithContentBody<T> ExecuteAndDeserializeWithContent<T>(Request m)
 
             return new ResponseWithContent<T>(response).DeserializeWithContent();
         }
+
+        private void logResponse(IRestResponse restResp)
+        {
+            if (restResp == null)
+            {
+                return;
+            }
+            try
+            {
+                var logDirPath = Path.Combine(Path.GetTempPath(), "GregLog");
+                if (!Directory.Exists(logDirPath))
+                {
+                    Directory.CreateDirectory(logDirPath);
+                }
+                using (StreamWriter outputFile = new StreamWriter(Path.Combine(logDirPath, Guid.NewGuid().ToString() + ".txt")))
+                {
+                    var logObj = new
+                    {
+                        timeStamp = DateTime.Now.ToString("MM/dd/yyyy HH:mm:ss"),
+                        respContent = restResp.Content,
```

Comment:
`respContent` should be public data right? nothing sensitive I hope..

Reply:
correct